src/toil/test/src/regularLogTest.py

```diff
@@ -91,5 +91,5 @@ def testRegularLog(self):
                                               './toilTest',
                                               '--clean=always',
```

Comment:
On an unrelated note, there have been issues in the past about weird stuff happening with '=' in command options that took arguments. It wouldn't hurt to just delete the = and add the argument as another item in the list.

Reply:
Change this if there's an issue for it.  This is also the wrong PR for this.